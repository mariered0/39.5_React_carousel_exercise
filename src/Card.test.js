import { render } from "@testing-library/react";
import Card from "./Card";

//smoke test
it("it renders without crashing", () => {
  render(<Card />);
});

//snapshot test
it("matches snapshot", () => {
  const { asFragment } = render(
    <Card caption="Beach" src="image1" currNum="1" totalNum="1" />
  );
  expect(asFragment()).toMatchSnapshot();
});
