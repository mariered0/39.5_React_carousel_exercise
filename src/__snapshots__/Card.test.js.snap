// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Beach
    </h4>
    <img
      alt="Beach"
      class="Card-image"
      src="image1"
    />
    <small
      class="Card-small"
    >
      Image 1 of 1.
    </small>
  </div>
</DocumentFragment>
`;
